closure/compiler/test/BUILD

```diff
@@ -33,7 +33,7 @@ closure_js_binary(
 
 file_test(
     name = "minification",
-    content = "console.log(\"hello world\");\n",
+    content = "(function(){console.log(\"hello world\");}).call(this);\n",
```

Comment:
I'm mostly just looking for a test that uses the new flag. It might be cool to demonstrate to the user how to disable the output wrapper entirely. Of the three you listed, which one is canonical? And if multiple ones do the same thing, can we have a fail() check that forces the user to choose only one of them?